.github/workflows/test.yml

```diff
@@ -91,8 +91,16 @@ jobs:
             cli/**
             packages/**
             shim/**
+            Cargo.*
+            .cargo/**
             !**.md
             !**.mdx
+      - name: Test changes
+        id: tests
```

Comment:
Maybe make it a little bit more specific, like `cli_tests` or `go_tests` or `repo_tests`